modules/UI/UI.js

```diff
@@ -40,6 +35,13 @@ import FollowMe from "../FollowMe";
 var eventEmitter = new EventEmitter();
 UI.eventEmitter = eventEmitter;
 
+/**
+ * Whether an overlay is visible or not.
+ * @type {boolean}
+ * @public
```

Comment:
I think this would be confusing because technically this variable is public. What about adding more comments?